### DISCUSSION TITLE
Can I use S3 as a persistent storage layer?

### DISCUSSION THREAD

**Discussion by pelegb** [ueberdosis/hocuspocus#532] [Q&A]
Hi,
Thanks for the great work.
I am interested to hear if I can use AWS S3 as a persistent storage for the documents.

Thanks again

**Comment by janthurau** [comment]
hey @pelegb, you can probably, if you use extension-database and just use s3 instead of a database. Not sure if that's a good idea, but feel free to give it a try (and let us know if it works :))
